dvc/commands/experiments/__init__.py

```diff
@@ -58,6 +58,15 @@ def add_parser(subparsers, parent_parser):
     hide_subparsers_from_help(experiments_subparsers)
 
 
+def add_keep_selection_flag(experiments_subcmd_parser):
+    experiments_subcmd_parser.add_argument(
+        "--keep",
+        action="store_true",
+        default=False,
+        help="Keep the selected experiments instead of removing them ",
```

Comment:
I don't think you need an extra space at the end? Also we probably have usually a `.` at then end.

Reply:
Sure. 
I'm pretty impressed by the level of detail of your reviews 🤩 Sorry for the distraction. I'll make sure to review my own code multiple times before pushing next time 🙄